### ISSUE TITLE
No cloud in Larastan constraints

### ISSUE THREAD

**Issue by szepeviktor** [nunomaduro/larastan#2134]
[Tests fail](https://github.com/larastan/larastan/actions/runs/12060071745/job/33629830845) because we are low on version constraints.

This is too new https://github.com/laravel/framework/pull/53623

**Comment by szepeviktor** [comment]
https://github.com/larastan/larastan/blob/b421a041af1a51e1b628476cb46826f7538a3055/.github/workflows/tests.yml#L69-L72

**Comment by canvural** [comment]
Yeah, I have no idea why it happens. Because it passes the first time, but when analyzed from outside of the directory it fails 🤯

**Comment by calebdw** [comment]
@szepeviktor, this can be closed now right?

**Comment by szepeviktor** [comment]
Yes, fixed in #2166
